includes/class-sensei-question.php

```diff
@@ -1185,11 +1185,12 @@ public static function multiple_choice_load_question_data( $question_data, $ques
 			}
 
 			// Merge right and wrong answers
+			$merged_options = [];
 			if ( is_array( $question_data['question_right_answer'] ) ) {
 
 				$merged_options = array_merge( $question_data['question_wrong_answers'], $question_data['question_right_answer'] );
 
-			} else {
+			} elseif ( is_array( $question_data['question_wrong_answers'] ) ) {
```

Comment:
I see there is a usage of `$question_data['question_wrong_answers']` as an array a few lines above.
What do you think about initializing it with an array if it is not an array?

Reply:
I just realized that in the current implementation we miss data if `question_wrong_answers` isn't an array but we still have a value inside `question_right_answer`.

Reply:
https://github.com/Automattic/sensei/pull/4491#discussion_r760857630